### PR TITLE
Code reviews

### PR DESCRIPTION
**Pull request by LassiAutioSolita**

### CODE REVIEW

File: _posts/2022-01-14-code-reviews.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Code reviews
```
Comment (LassiAutioSolita) [inline _posts/2022-01-14-code-reviews.md:3]: I am not confident with the title. Is it too general? Or is it good to be general?

Alternatives that come to my mind
* Code reviews are important!
* Code reviews are even more important than programming itself
* Motivations and tips for code reviews

Somehow I want to say something specific here, but content of this blog post is also quite general or wide range.

File: _posts/2022-01-14-code-reviews.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Reviewing finished code is too late - and other tips and motivations for code reviews
```
Comment (LassiAutioSolita) [inline _posts/2022-01-14-code-reviews.md:3]: Changed the title according to feedback. Tried to put one thing from the blog post to here ("Reviewing finished code is too late"). But as it is not the only thing in the blog post, put general part after - ("and other tips and motivations for code reviews").

Comment (denzilferreira) [reply _posts/2022-01-14-code-reviews.md:3]: On the other hand, code is "never finished" :) Maybe something like: There is never a perfect time to review your code: do it!

Comment (filipwastberg) [reply _posts/2022-01-14-code-reviews.md:3]: I kind of liked you're first proposal `Code review is more important than progamming`

File: _posts/2022-01-14-code-reviews.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+author: lassiautio
+excerpt: >
+  Why should we do code reviews? What makes code reviews even more important than programming? Why and how to shift-left code reviews?
+  This blog post will give answers to these questions and go deeper into the world of code reviews.
+  In the end, there will be also some tips on why code reviews should be small, and why you shouldn't comment too much on code reviews.
+tags:
+ - code review
+---
+
+Code reviews are something we all have been involved, and what we do continuously in our projects. We might have contradictory feelings about them. Often they feel like just bureaucracy that is required by a manager. Sometimes we want someone to give us feedback on our code. When we see poor code, we might think that we should've reviewed the code better.
```
Comment (filipwastberg) [inline _posts/2022-01-14-code-reviews.md:13]: involved *in*

Comment (filipwastberg) [inline _posts/2022-01-14-code-reviews.md:13]: we might have *mixed* feeling 

I think sounds better than *contradictory*

File: _posts/2022-01-14-code-reviews.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+This is a blog post version from my presentation that I had at [DevDay 2021](https://dev.solita.fi/2021/12/13/devday-of-solita.html).
+
+# Reasons to do code reviews
+
+## Learning and sharing knowledge
+
+This might be a little surprising for someones, but reducing bugs or code quality are not the most important reasons to do code reviews. It is sharing the knowledge. Code reviews are meant to share knowledge about the code base and also programming in general.
+
+"Oh, I didn't know it can also be done this way. I will use it next time!" This is something I've said many times when I've reviewed code.
```
Comment (filipwastberg) [inline _posts/2022-01-14-code-reviews.md:25]: Have you "said it" or thought to yourself? :D Maybe:

Is something I've thought to myself many times when doing code review.

File: _posts/2022-01-14-code-reviews.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+Why? How about feature B that I'm now programming? It is a super important feature. I will review D after I've done B.
+
+The reason is simple: the feature D is **closest to be done** and thus has the highest value after feature C.
+
+## Make an impact and do the code review soon
+
+*Only done work has real value (€$£)*. All cards in an agile board that are not yet done don't have any value yet. They will have value when they are done (and deployed to the production).
+
+In-review is close to being done. It will soon create value sooner than cards in todo or in-progress columns. The more right you go in an agile board, the sooner it will have real value. Thus *things on right are more important than things on left*. Help your co-worker to finish his/her task and create value by reviewing the code as soon as possible.
```
Comment (filipwastberg) [inline _posts/2022-01-14-code-reviews.md:71]: To many soons?

**Review by filipwastberg — COMMENTED** [review]
Just some small corrections! Nice post and important topic :)

File: _posts/2022-01-14-code-reviews.md (hunk 3 above)
Comment (LassiAutioSolita) [reply _posts/2022-01-14-code-reviews.md]: Thanks! I used Google translator to find a word from Finnish. But mixed is the one that I actually was searching for.

File: _posts/2022-01-14-code-reviews.md (hunk 5 above)
Comment (LassiAutioSolita) [reply _posts/2022-01-14-code-reviews.md]: True. At least one "soon" fewer now.

File: _posts/2022-01-14-code-reviews.md (hunk 2 above)
Comment (LassiAutioSolita) [reply _posts/2022-01-14-code-reviews.md]: Sorry for late reply. I also liked my original "code review is more important than programming" even if I wasn't sure about it. So I changed to it. Many good options for the title.

**Review by crystoll — APPROVED** [review]

**Pull request merged**